handbook/data.md

```diff
@@ -275,6 +275,45 @@ The following is the description of the air pollution exposure data.
 ```
 `````
 
+### PM2.5 US High Resolution Grid, 2000-2016
+
+`````{dropdown} PM2.5 US Grid
+```{list-table}
+:header-rows: 0
+
+* - spatial_coverage
+  - US
+* - spatial_resolution
+  - 1km x 1km
+* - temporal_coverage
+  - 2000-2016
+* - temporal_resolution
+  - annually
+* - size
+  - ~80 MB/year
+* - fasse_location
+  - `/n/dominici_nsaph_l3/Lab/data/exposure/pm25/whole_us/annual/grid_pts/qd_new_predictions`.
```

Comment:
Is this the location of the original file or the final (output) file?

Reply:
This is the location of the folder with the data files I described. What do you mean by original?
I haven't created this dataset; @audiracmichelle shared me the directory.

Reply:
Ok, that's good, just checking!